### PR TITLE
Ros lisp script

### PR DESCRIPTION
**Pull request by tmccombs**
Here is an initial implementation using roswell (https://github.com/roswell/roswell) I think this is much simpler than using CIM, and running the tests can be simplified with roswell scripts (such as the run-prove roswell script for the prove framework).

### CODE REVIEW

File: lib/travis/build/script/lisp.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+module Travis
+  module Build
+    class Script
+      class Lisp < Script
+        DEFAULTS = {
+          lisp: 'sbcl'
+        }
+
+        ROS_URL = "https://raw.githubusercontent.com/snmsts/roswell/release/scripts/install-for-ci.sh"
```
Comment (snmsts) [inline lib/travis/build/script/lisp.rb]: I'd recommend "https://raw.githubusercontent.com/roswell/roswell/release/scripts/install-for-ci.sh"

**Comment by KeenS** [comment]
I'd like to have name 'common-lisp' rather than 'lisp'

**Comment by snmsts** [comment]
CommonLisp ?

**Comment by KeenS** [comment]
Looks good to me

**Comment by sionescu** [comment]
I committed and squashed the previous commits, since they are useless. I think we're ready for an initial pull request upstream.

**Pull request closed without merging**